### DISCUSSION TITLE
v1.0

### DISCUSSION THREAD

**Discussion by tianpf** [spatial-go/geoos#21] [General]
Reconstruction of geometric coverage, spatial analysis, vector construction and other related methods
No need to install GeoS geometric calculation library, independent implementation of space calculation methods

<hr /><em>This discussion was created from the release <a href='https://github.com/spatial-go/geoos/releases/tag/v1.0'>v1.0</a>.</em>

**Comment by xiaogu-space** [comment]
Congratulations on the milestone release

**Reply by tianpf** [reply]
Crossing the Chishui River Four Times.

**Comment by pipipizhuqaq** [comment]
Great moment！
